examples/hybrid_jobs/0_Creating_your_first_Hybrid_Job/0_Creating_your_first_Hybrid_Job.ipynb

```diff
@@ -0,0 +1,623 @@
+{
```

Comment:
<blockquote>The next step is to run this on a real QPU to see how well the simple qubit rotation works.</blockquote><p>The next step is to see how well the qubit rotation works on a real QPU.</p><p> </p><blockquote>This is because hybrid jobs have a separate queue from standalone tasks</blockquote><blockquote>However, hybrid jobs have a separate queue from standalone tasks</blockquote><p>This is very confusing because we are referring to two separate concepts with the same formulation. The first one, refer to the priority task queue, i.e., every task sent from a job has priority. The second one refers to the jobs queue, i.e. your jub might not start immediate but have to wait until no other job is runnign</p><p> </p><blockquote><strong>Note:</strong> Only hybrid jobs running on AWS receive priority. Hybrid jobs running locally, or with a mismatched device ARN do not get priority task queueing.   </blockquote><p>mismatched between what and what? Also, need to link to documentation for local jobs - otherwise this comes out of the blue, readers will not know what that is</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>